serpentTools/parsers/branching.py

```diff
@@ -15,6 +15,164 @@ class BranchingReader(XSReader):
 
     def __init__(self, filePath):
         XSReader.__init__(self, filePath, 'branching')
+        
+    def parse_coe(self, coefile, grabB1=True):
```

Comment:
As I mentioned in the PR comment, the branching reader has been nearly completed [here](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/branching/serpentTools/parsers/branching.py#L27).

Reply:
OK, I'll remove this. Thanks!